packages/docs/components/modal.md

```diff
@@ -61,33 +82,41 @@ Here are a few smells for when a modal dialog may be inappropriate:
 
 </Visual>
 
-## Accessibility
+### Use when...
 
 <Description>
 
-Modal dialogs are notorious for their accessibility issues, but don't worry! We've got your back.
+- Blocking user interaction for security reasons - e.g. "Please reenter your password to initiate launch."
+- Gathering immediate information to continue a task - e.g. "A hangar number is required to continue."
+- Confirming user interaction during destructive actions - e.g. "Please confirm the starship name to begin disassembly."
+- Offering an alternative that would lessen the user's required effort - e.g. "Additional security clearance will be required - are you sure you want to continue?"
 
 </Description>
 
-### Accessible attributes
+### Don't use when...
 
 <Description>
 
-The modal components makes use of several ARIA attributes in order to give assistive technologies (AT) as much information as possible.
-
-- `aria-hidden` - This indicates whether or not the modal is visible and should be automatically set to `true` or `false` by our companion Javascript.
-- `role="dialog"` - Until the `role` element is widely supported, this attribute indicates the semantic status of the modal to the browser. Please use `"alertdialog"` instead if your modal represents a dangerous action or error.
-- `aria-modal="true"` - This indicates that the rest of the workflow is stopped while this element is present.
-- `aria-labelledby="ods-modal--title-id"` - This attribute informs ATs which other element can be treated as the label for this dialog. Please use the unique `id` associated with the dialog's title.
-- `aria-label="Close"` - Please use this tag on any `button` that allows the user to exit the dialog. By default, this is included on the modal's exit "X".
+- Immediate action is not required
+- The dialog would interrupt a workflow
+- Additional context may be required or helpful
+- The dialog content is long-form
+- The user has not triggered the modal via interaction (e.g. on page load)
+- There is already a modal being displayed
 
 </Description>
 
+## Content guidelines
+
+Context is key. There needs to be a direct connection between the trigger (i.e. a button or link) and the modal that follows. Repeating the trigger's copy within the modal can help reinforce this connection. Providing a sense of connection between the trigger and modal is important to ensure a consistent user experience.
+
+## Accessibility
+
 ### Accessible behavior
 
 <Description>
 
-When a modal dialog is opened, interaction is limited to the new context. While this may seem expected, it can surprise users - and is even worse for users requiring assistive technologies.
+When a user opens a modal, interaction is limited to the new context. While this may seem expected, it can surprise users. This is heightened for users requiring assistive technologies.
```

Comment:
```suggestion
When a user opens a modal, interaction is limited to the new context. While this may seem expected, it can surprise users. This is heightened for users requiring assistive technologies as the focus is kept with the bounds of the modal element.
```

Reply:
We note this above and below. I left it out here since there are additional reasons this may be surprising (lack of visual context, etc). That work?